brave-snouts-frontend/src/app/features/auction-feature/user/user-auctions/user-items.component.html

```diff
@@ -7,7 +7,7 @@
         'mt-4': mediaObs.isActive('lt-sm')
     }" >Praćeni predmeti</div>
     
-    <virtual-scroller #scroll [items]="(trackedItems$ | async)" *ngIf="(trackedItems$ | async)?.length > 0" [parentScroll]="scrollingBlock">
+    <virtual-scroller class="full-items-grid" #scroll [items]="(trackedItems$ | async)" *ngIf="(trackedItems$ | async)?.length > 0" [parentScroll]="scrollingBlock">
```

Comment:
please add [bufferAmount] here like on `items-list` additionally. Since we'r doing changes here...